perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -23,9 +23,10 @@
 	 * @param member
 	 * @throws InternalErrorException
 	 * @throws MemberAlreadyRemovedException
-	 * @throws GroupOperationsException
+	 * @throws WrongAttributeValueException
+	 * @throws WrongReferenceAttributeValueException
 	 */
-	void deleteMember(PerunSession sess, Member member) throws InternalErrorException, MemberAlreadyRemovedException, GroupOperationsException;
+	void deleteMember(PerunSession sess, Member member) throws InternalErrorException, MemberAlreadyRemovedException, WrongAttributeValueException, WrongReferenceAttributeValueException;
```

Comment:
Please remove ``WrongAttributeValueException`` and ``WrongReferenceAttributeValueException``, they are not actually thrown in Bl of this method. After removal, you can remove it also from ``deleteAllMembers()`` and also ``deleteUser()``